src/js/components/Select/stories/ClearValue.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Clear } from 'grommet-icons/icons/Clear';
+
+import { Box, Grommet, Select, Button } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Text } from '../../Text';
+
+const optionList = [
+  'One',
+  'Two',
+  'Three',
+  'Four',
+  'Five',
+  'Six',
+  'Seven',
+  'Eight',
+  'Nine',
+  'Ten',
+];
+
+const ClearSelection = () => {
+  const [value, setValue] = useState('');
```

Comment:
give each select a different value in state. Interacting with the example is a little jarring when all three selects change at the same time.